pegasus/sites.v3/code.org/views/tabs_section/resources/resources_tabs.haml

```diff
@@ -51,4 +51,4 @@
     %summary=hoc_s(:"resources_tabs.progress.tab_label")
     = view :"tabs_section/resources/resources_tabs_content_progress"
 
-%script{src: '../js/tabs-section.js'}
+= view :tabs_section
```

Comment:
I confirmed this works on the Resources tab sections seen on https://code.org/teach, etc.